apps/gauzy/src/app/pages/integrations/components/plugin-builtin/plugin-builtin.component.html

```
@@ -0,0 +1,102 @@
+					[class.coming-soon]="integration.isComingSoon"
+					[class.clickable]="!integration.isComingSoon"
+					(click)="navigateTo(integration)"
+					(keydown.enter)="navigateTo(integration)"
+					[attr.tabindex]="integration.isComingSoon ? null : 0"
+					[attr.role]="integration.isComingSoon ? null : 'button'"
+					[nbTooltipDisabled]="!integration.isComingSoon"
+					[nbTooltip]="'INTEGRATIONS.COMING_SOON' | translate"
+				>
+					<div class="card-logo">
+						<img
+							[src]="integration.fullImgUrl"
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)